CONTRIBUTING.md

```diff
@@ -148,6 +148,7 @@ change. Common tags are:
   * PERF for performance optimizations
   * PRJ for project creation changes 
   * ACT for actions
+  * DBG for debugger
```

Comment:
Btw, this section now uses **a lot** of space, so maybe hind it behind `\<details\>\</details\>`?